### PR TITLE
[DSY-1273] Updates nat button to allows icons

### PR DESCRIPTION
**Pull request by vitorpirestw**
# Description

- Updates NatButton to allow icons on the right and left position

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: Sources/Public/Components/Button/NatButton+Icon.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+extension NatButton {
+    /**
+     IconSide is a enum that represents the icon side options for the button component.
```
Comment (raonivaladares) [inline Sources/Public/Components/Button/NatButton+Icon.swift:3]: IconSide ->  Position

**Review by raonivaladares — APPROVED** [review]
👍

File: Sources/Public/Components/Button/NatButton+Icon.swift (hunk 1 above)
Comment (vitorpirestw) [reply Sources/Public/Components/Button/NatButton+Icon.swift]: Done!

**Comment by vitorpirestw** [comment]
Thanks!

**Pull request merged**